api/book.go

```diff
@@ -10,10 +11,12 @@ import (
 )
 
 type Book struct {
-	Title   string
-	Author  string
-	Self    *Link
-	History []Checkout
+	Id       string
```

Comment:
I would suggest adding a JSON mapping for each field of this model so that we can return the items in lower case (according to swagger spec).
For example:
```
...
 ID string `json:"id"`
 ...
```
would generate a JSON with 'ID' mapped to 'id'.

Reply:
ohhhh 😮 - I didn't know this was a thing! Thank you